content/docs/5.1/concepts/explore-tikv-features/fault-tolerance.md

```diff
@@ -1,10 +1,220 @@
 ---
-title: Fault tolerance
-description: TODO
+title: Fault Tolerance and Recovery
+description: Learn how TiKV recovers from failures
 menu:
     "5.1":
         parent: Features
         weight: 2
 ---
 
-TODO
+This page walks you through a simple demonstration of how TiKV recovers from failures and continues to provide services during failures.
+
+1. Starting a 6-node local cluster.
+2. Run a sample workload via [go-ycsb](https://github.com/pingcap/go-ycsb), terminate a node to simulate a failure, and see how the cluster continues uninterrupted.
```

Comment:
```suggestion
2. Run a sample workload via [go-ycsb](https://github.com/pingcap/go-ycsb), then kill a node to simulate a failure, and see how the cluster seamlessly continues.
```

Reply:
Ditto.

Reply:
This is also for uniformity. You used the word `kill` in the following step-by-step instruction, so it'll be better to use the same word for the same action. Besides, `seamlessly` sounds more stable than `uninterrupted` which is like 'TiKV will not shutdown but the performance may drop'.

Reply:
I am sorry, I can't see the difference. :cry:

Reply:
kill vs terminate? no difference?

Reply:
Done.